README.md

```diff
@@ -325,13 +325,27 @@ end
 ## Releasing
 
 After following the steps from the [Developing section](#developing) to build
-and test the gem, and merging all changes to the `master` branch, first bump
-the version number by editing
-[`lib/jekyll_pages_api_search/version.rb`](lib/jekyll_pages_api_search/version.rb).
-Then run `bundle exec rake release` to build the gem, tag the head commit in
-`git`, push the branch and tag to GitHub, and ultimately push the release to 
-[`jekyll_pages_api_search` on
-RubyGems.org](https://rubygems.org/gems/jekyll_pages_api_search).
+and test the gem: 
+
+1. Ensure all changes for the release have already been merged all into the
+   `master` branch.
+   
+1. Bump the version number by editing
+   [`lib/jekyll_pages_api_search/version.rb`](lib/jekyll_pages_api_search/version.rb).
+
+1. Commit the version number update directly to the `master` branch, replacing
+   `X.X.X` with the new version number:
+   ```sh
+   $ git add lib/jekyll_pages_api_search/version.rb
+   $ git commit -m 'Bump to vX.X.X'
+   ```
```

Comment:
`git add` is the command to stage changes for `git commit`. http://git-scm.com/documentation is a good intro to git in general.

It would be preferable for me to cut v0.4.4 now, so I can get apps.gov to depend on it directly. No other sites will be affected, and you can wait until Monday to push it to 18F/guides-template yet so most of the Pages sites will get the new JavaScript (which is functionally the same as the 0.4.3, just refactored).